Noti/PushManager.swift

```diff
@@ -460,6 +422,118 @@ class PushManager: NSObject, WebSocketDelegate, NSUserNotificationCenterDelegate
         
     }
     
+    // Given a push JSON object from Pushbullet create a notification
+    // Used in mirror and subtipe:push
+    internal func createNotification(msg : JSON) -> NSUserNotification{
+        var push = checkEncryption(message: msg)                            // If it's encrypted, decrypt it
+        let notification = NSUserNotification()
+        if(push == JSON.null || push["dismissed"].boolValue){               // If somenthing went wrong or it's already dismissed
+            return notification
+        }
+        notification.actionButtonTitle = "Show"
+        if let type = push["type"].string {
+            switch type {
+            case "url":                                                     // Special type of Noti: url type
+                notification.title = "Url"                                  // We have no title here
+                notification.informativeText = push["url"].string
+                notification.identifier = push["iden"].string
+                break;
```

Comment:
swift does not fallthrough (like other languages), so this break statement is unneeded